Code/GraphMol/MolEnumerator/MolEnumerator.h

```diff
@@ -237,6 +240,55 @@ class RDKIT_MOLENUMERATOR_EXPORT RepeatUnitOp : public MolEnumeratorOp {
   void initFromMol();
 };
 
+using stereo_flipper_t = std::shared_ptr<StereoFlipper>;
```

Comment:
We don't use snake_case. Please change this something in CamelCase. Maybe `StereoFlipperSP`?

Reply:
fixed!